### ISSUE TITLE
[A11y] Logo alt-Title should display "Back to Homepage"

### ISSUE THREAD

**Issue by Molochem** [kitconcept/volto-light-theme#336]
When looking at the alt-Title of the Logo-Image, it displays hardcoded "Plone-Site"

**To Reproduce**
Hover the Page-Logo or Inspect the Logo with the Browser-Inspector

**Expected behavior**
It should be displaying something like "Back to Homepage" to make it a11y-compliant and Screenreader-accessible

**Screenshots**
![image](https://github.com/kitconcept/volto-light-theme/assets/140883099/b32a90d2-bdf8-49b8-acd5-ef99bd714c75)
On Hover

![image](https://github.com/kitconcept/volto-light-theme/assets/140883099/47c019c6-bcbb-4cb7-9feb-f5096fa91e56)
On Inspect

**Comment by sneridagh** [comment]
@davisagli I think that we backported and update the VLT Logo shadow with the current state of the Logo component and all the decisions taken in there to VLT 6.
Could you please take a look and confirm that now is correct, or state what's missing?

**Comment by davisagli** [comment]
@sneridagh It's close but we need to remove the `title` attribute. It's not helpful to show visual users a tooltip pointing out that they are looking at a logo, but the tooltip gets in the way of the logo. :)
